config/templates/abandonWarning.md

```diff
@@ -1,5 +1 @@
-Hello @{assignee}, you have been unassigned from this issue because you have not updated this issue or any referenced pull requests for over {total} days.
-
-You can reclaim this issue or claim any other issue by commenting `@{username} claim` on that issue.
-
-Thanks for your contributions, and hope to see you again soon!
+@{assignee} you have been unassigned from this issue because you have not made any updates for over {total} days. Please feel free to reclaim the issue if you decide to pick up again. Thanks!
```

Comment:
This should be either “@{assignee}, you …” with a comma or “@{assignee} You …” with a capital letter. (Similarly for other strings.)